hw06-ATM/src/main/java/ru/AtmImpl.java

```diff
@@ -0,0 +1,93 @@
+package ru;
+
+import java.util.*;
+
+public class AtmImpl implements Atm {
+    private HashSet<BanknoteCell> banknotes = new HashSet<BanknoteCell>();// переделать на TreeSet
+
+    public AtmImpl(List<Denomination> listCells) {
+        listCells.forEach(denomination -> this.banknotes.add(new BanknoteCell(denomination, 0)));
+    }
+
+    @Override
+    public HashSet<BanknoteCell> putBanknotes(Integer sum) throws RuntimeException {
```

Comment:
```suggestion
    public HashSet<BanknoteCell> putBanknotes(Integer sum) {
```

` throws RuntimeException` здесь совсем не нужен

Reply:
поняла. спасибо